cmd/notary/keys.go

```diff
@@ -540,7 +531,11 @@ func (k *keyCommander) keyPassphraseChange(cmd *cobra.Command, args []string) er
 	// unlocking passphrase and reusing that.
 	passChangeRetriever := k.getRetriever()
 	keyStore, err := trustmanager.NewKeyFileStore(config.GetString("trust_dir"), passChangeRetriever)
-	err = keyStore.AddKey(filepath.Join(keyGUN, keyID), role, privKey)
+	keyInfo, err := cs.GetKeyInfo(keyID)
+	if err != nil {
+		return err
+	}
+	err = keyStore.AddKey(filepath.Join(keyInfo.Gun, keyID), role, privKey)
```

Comment:
I think that's a good idea. I'm currently working on what the API change would look like for each usage -- so far it seems that importing will have the biggest internal changes, but it should be good.